examples/webgpu/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Bouncy Ball - Comparing Web Animation Techniques - WebGPU</title>
+  <link rel="stylesheet" href="../shared-styles.css">
+  <style>
+  body,
+  html,
+  #canvas {
+    height: 100%;
+    width: 100%;
+    margin: 0;
+  }
```

Comment:
This is incomplete CSS. You can see the issue here: 

https://sparkbox.github.io/bouncy-ball/examples/webgl/index.html

There's a scrollbar.  Add the `display: block;` and the canvas goes from `inline` to `block` and the scrollbar disappears


```suggestion
    display: block;
  }
```

Reply:
Done